launcher/Cargo.toml

```diff
@@ -8,7 +8,7 @@ description = "Text Generation Launcher"
 [dependencies]
 clap = { version = "4.5.1", features = ["derive", "env"] }
 ctrlc = { version = "3.4.2", features = ["termination"] }
-nix = "0.27.1"
+nix = { veersion = "0.28.0", features = ["process", "signal"] }
```

Comment:
```suggestion
nix = { version = "0.28.0", features = ["process", "signal"] }
```

Reply:
oh lol. Unclear how that passed locally 🤔🤔🤔